js_modules/dagster-ui/packages/ui-core/src/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -61,14 +61,15 @@ export const SidebarAssetInfo = ({graphNode}: {graphNode: GraphNode}) => {
   const {lastMaterialization} = liveData || {};
   const asset = data?.assetNodeOrError.__typename === 'AssetNode' ? data.assetNodeOrError : null;
 
-  const recentEvents = useRecentAssetEvents(
-    asset?.assetKey,
-    1,
-    MaterializationHistoryEventTypeSelector.MATERIALIZATION,
+  const recentEvents = useRecentAssetEvents(asset?.assetKey, 1, [
+    AssetEventHistoryEventTypeSelector.MATERIALIZATION,
+    AssetEventHistoryEventTypeSelector.OBSERVATION,
+  ]);
+  const materializations = recentEvents.events.filter(
+    (event) => event.__typename === 'MaterializationEvent',
   );
-  const latestMaterializationEvent = recentEvents.materializations
-    ? recentEvents.materializations[recentEvents.materializations.length - 1]
-    : undefined;
+
+  const latestMaterializationEvent = materializations[materializations.length - 1]; // TODO shouldn't this be the first one in the list, not last? I thought the list was descending timestamp (newer first)
```

Comment:
this stumped me a bit - to me it looks like the existing observation/materialization resolvers return event in descending order (newest first), but this would select the oldest of the returned events. Is that a bug or am i missing something about the ordering

Reply:
Hmmm, I'm not entirely sure, you could try console.logging without your changes to verify. cc @bengotow

Reply:
i think it works because we only fetch 1 event